From: Michele Locati <michele@locati.it>
Date: Thu, 30 Jul 2020 18:58:02 +0200
Subject: [PATCH] build: Fix build failure on mingw (formatstring_ruby)

This fixes the following issue when compiling for Windows:
undefined reference to `_imp__formatstring_ruby'

--- a/gettext-tools/woe32dll/gettextsrc-exports.c
+++ b/gettext-tools/woe32dll/gettextsrc-exports.c
@@ -50,6 +50,7 @@ VARIABLE(formatstring_python)
 VARIABLE(formatstring_python_brace)
 VARIABLE(formatstring_qt)
 VARIABLE(formatstring_qt_plural)
+VARIABLE(formatstring_ruby)
 VARIABLE(formatstring_scheme)
 VARIABLE(formatstring_sh)
 VARIABLE(formatstring_smalltalk)
